unit_test/test_g2c_degrib2c_grib2.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+#            Test G2C Library 
+#    This script uses to test G2C library with following routines:
+#    - seekgb.c ; fseek; fread; g2_info; g2_getfld; g2_free;
+#      g2_unpack1; g2_unpack2; g2_unpack3, g2_unpack4, g2_unpack5;
+#      g2_unpack6; g2_unpack7
+
+set +x
+
+echo " "
+echo "###################################################"
+echo " Testing G2C Library "
+echo "###################################################"
+echo " "
+
+export dir=` pwd `
+
+. $MODULESHOME/init/bash
```

Comment:
Remove all mentions of modules. Modules must be loaded correctly before the test is run.